src/hooks/useUniswapAnjPrice.js

```diff
@@ -7,14 +7,14 @@ import { Client } from 'urql'
 const RETRY_EVERY = 3000
 
 const UNISWAP_URL = 'https://api.thegraph.com/subgraphs/name/lutter/uniswap-v2'
-const ETH_ANJ_PAIR = '0x0ffc70be6e2d841e109653ddb3034961591679d6'
+const ETH_ANT_PAIR = '0x0ffc70be6e2d841e109653ddb3034961591679d6'
```

Comment:
We should update the pair address for the pair. Note also that it should also be `ANT-ETH`

```suggestion
const ANT_ETH_PAIR = '0x9def9511fec79f83afcbffe4776b1d817dc775ae'
```

Reply:
ah no, i forgot to delete this file, since i thought that maybe we could just use this one now https://github.com/aragon/protocol-dashboard/blob/0a97474b7f125414014dbeb592f25dd7dc00f141/src/hooks/useTokenAmountToUsd.js#L20

and i changed from all the sites where `useANJAmountToUsd` was being used to start using this one. What do you think?

Reply:
Ah yes, we could use `useTokenAmountToUsd` as we already did. We would just need to make sure that the 0x api supports the new ANT token.

Reply:
cc @sohkai we need to do something with the 0x to let them know about the new token?